fedmsg_meta_fedora_infrastructure/autocloud.py

```diff
@@ -85,7 +86,10 @@ def handle_v2_autocloud_subtitle(self, msg, **config):
                                 compose_id=compose_id)
 
         if "autocloud.compose" in msg['topic']:
-            compose_id = msg['msg']['id']
+            compose_id = msg['msg'].get('id', '')
+
+            if compose_id is '' and "complete" in msg['topic']:
+                compose_id = msg['msg'].get('compose_id', '')
 
```

Comment:
The chances are very less (can say nil) that both are missing. Maybe we can add a default text for compose_id as `A compose`.

This code is under another check above if `autocloud.compose` is in the topic. So the resultant check would be like `autocloud.compose` and `complete` in the topic. 
I can change `complete` to `autocloud.compose.complete`